pkgs/buildNpmPackage/mkcache.js

```diff
@@ -0,0 +1,38 @@
+const assert = require('assert')
+const fs     = require('fs')
+const path   = require('path')
+const {promisify} = require('util')
+
+// find pacote from npm dependencies
+module.paths.push(path.join(process.argv[0], "../../lib/node_modules/npm/node_modules"))
+const pacote = require('pacote')
```

Comment:
is this some kind of joke name? pacote? really? I guess it's only funny because Spanish.